docs/building.md

```diff
@@ -13,10 +13,14 @@ $ cd betwixt
 $ npm install
 ```
 
-### Running
+### Running (regular proxy mode)
 
 You can run Betwixt right away using `npm start`.
 
+### Running with binding options and reverse proxy mode
+
+`npm start -- --proxy-port=8010 --proxy-host=0.0.0.0 --reverse-proxy=http://127.0.0.1:8081`
```

Comment:
Sorry for taking so much time to answer. :(

Nope, it is not a typo. They're necessary to propagate the command line arguments to electron.

```
$ npm start -- --proxy-port=8010 --proxy-host=0.0.0.0 --reverse-proxy=http://127.0.0.1:8081

> betwixt@0.0.0 start /home/mau/dev/betwixt
> electron main.js "--proxy-port=8010" "--proxy-host=0.0.0.0" "--reverse-proxy=http://127.0.0.1:8081"
```

From https://docs.npmjs.com/cli/run-script:
> As of npm@2.0.0, you can use custom arguments when executing scripts. The special option -- is used by getopt to delimit the end of the options. npm will pass all the arguments after the -- directly to your script